### ISSUE TITLE
[Feature Request] Make Mobile-friendly view customizable

### ISSUE THREAD

**Issue by bungabunga** [brave/browser-android-tabs#207]
As Firefox for Android falls behind Chromium/Brave security/privacy-wise it still has some neat features that i really miss on Brave/Chromium. One of them is the customization of it's Reading mode which is FF's way of Mobile-friendly view. You can change font-sizes and select between day and night mode.

As a person with sight problems the Android system-wide customizations of font sizes are not enough for me and the night mode is much more eye-friendly when reading long texts at night.

Please consider including something like this in Brave for Android.

![screenshot_20170406-200429](https://cloud.githubusercontent.com/assets/2994878/24769236/1048589a-1b06-11e7-84cd-31f7c508616c.png)
![screenshot_20170406-200438](https://cloud.githubusercontent.com/assets/2994878/24769235/1048cc44-1b06-11e7-8a51-86cb56e8934c.png)

**Comment by iamVP7** [comment]
Similar request made by another user in community website

[Firefox-like Read-Mode in brave](https://community.brave.com/t/firefox-like-read-mode-in-brave/2121)

**Comment by yodrew** [comment]
Yes, Firefox-like reader mode would be wonderful.

**Comment by recursionbane** [comment]
Yes, please. This is the main thing keeping me from using Brave by default.

**Comment by oraculix** [comment]
Reader Mode in Firefox is _the_ killer feature for me. I do use Brave on PC and Android and love it, but when it comes to reading articles in the evening, I will stick to Firefox as long as this is the only browser with such a feature. That means Brave is missing out on where most of my private attention goes, and according to many ratings I've read on Google Play, it's the same for many, many other users. And this is sad, because it touches one of Brave's main innovations: using attention to support creators.

My own personal guess is, that implementing Reader Mode should be relatively easy, and the ROI in terms of users finally converting completely to Brave will be huge. So, please, treat that request with higher priority!
